test/support/protocol_utils.rb

```diff
@@ -149,43 +124,32 @@ def req_finish
     def req_set_exception_breakpoints
       case ENV['RUBY_DEBUG_TEST_UI']
       when 'vscode'
-        send_request 'setExceptionBreakpoints',
+        send_dap_request 'setExceptionBreakpoints',
                       filters: [],
                       filterOptions: [
                         {
                           filterId: 'RuntimeError'
                         }
                       ]
-        res = find_crt_dap_response
-        assert_dap_response :SetExceptionBreakpointsResponse, res
       when 'chrome'
-        send_request 'Debugger.setPauseOnExceptions',
-                      state: 'all'
-        res = find_crt_cdp_response
-        assert_cdp_response 'Debugger.setPauseOnExceptions', res
+        send_cdp_request 'Debugger.setPauseOnExceptions', state: 'all'
       end
     end
 
     def req_step_back
       case ENV['RUBY_DEBUG_TEST_UI']
       when 'vscode'
-        send_request 'stepBack',
-                      threadId: 1
-        # TODO: Return the response for "stepBack"
-        # res = find_crt_dap_response
-        # assert_dap_response :StepBackResponse, res
+        send_dap_request 'stepBack', threadId: 1
       end
     end
 
     def req_terminate_debuggee
       case ENV['RUBY_DEBUG_TEST_UI']
       when 'vscode'
-        send_request 'terminate'
-        assert_disconnect_result
+        send_dap_request 'terminate'
+        cleanup_reader
       when 'chrome'
-        send_request 'Runtime.terminateExecution'
-        res = find_crt_cdp_response
-        assert_cdp_response 'Runtime.terminateExecution', res
+        send_cdp_request 'Runtime.terminateExecution'
```

Comment:
Maybe, I forgot to add `cleanup_reader` method. Could you add it here, too?

Reply:
Updated 👍